alicloud/resource_alicloud_cs_swarm_test.go

```diff
@@ -101,6 +101,31 @@ func TestAccAlicloudCSSwarm_vpc(t *testing.T) {
 	})
 }
 
+func TestAccAlicloudCSSwarm_vpc_zero_node(t *testing.T) {
+	var container cs.ClusterType
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+
+		IDRefreshName: "alicloud_cs_swarm.cs_vpc",
+
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckSwarmClusterDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCSSwarm_basic_zero_node,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckContainerClusterExists("alicloud_cs_swarm.cs_vpc", &container),
+					resource.TestCheckResourceAttr("alicloud_cs_swarm.cs_vpc", "node_number", "0"),
+					resource.TestCheckResourceAttr("alicloud_cs_swarm.cs_vpc", "nodes.#", "0"),
```

Comment:
Please check all attributes?

In addition, can you add another step to test updating node number from 0 to 2?

Reply:
done.

Reply:
yes, I know. So, it should check disk size is 0 and disk type is "".

In addition, can you add another step to test updating node number from 0 to 2?

Reply:
done. also already added additional unit tests (called "TestAccAlicloudCSSwarm_vpc_zero_node_update") to check updating node number from 0 to 2:
=== RUN   TestAccAlicloudCSSwarm_vpc_zero_node
--- PASS: TestAccAlicloudCSSwarm_vpc_zero_node (357.06s)
=== RUN   TestAccAlicloudCSSwarm_vpc_zero_node_update
--- PASS: TestAccAlicloudCSSwarm_vpc_zero_node_update (693.68s)
PASS
ok	github.com/terraform-providers/terraform-provider-alicloud/alicloud	1050.74